src/libpmemfile/preload.c

```diff
@@ -92,18 +92,19 @@
 #define SYS_pwritev2 328
 #endif
 
+#define MAX_SYSCALLS 0x200
```

Comment:
This is not a big deal, but MAX_SYSCALLS seems to somehow suggest, that 0x200 syscalls are handled, and we stop after that, or whatever.
This constant is not about some sort of "upper bound count of syscalls".
 
Perhaps MAX_SYSCALL_NUMBER .

This is meant to be the largest syscall number we care about, which is just an id of a syscall, not any kind of count.
It is used while specifying array sizes, but only because those arrays are indexed by syscall numbers. Incidentally, in that case the arrays should be declared as:
`static bool syscall_number_filter[MAX_SYSCALL_NUMBER + 1];`


But I can live without this.